commands.json

```diff
@@ -1217,19 +1217,16 @@
   },
   "HSET": {
     "summary": "Set the string value of a hash field",
-    "complexity": "O(1)",
+    "complexity": "O(N) where N is the number of fields being set",
```

Comment:
One final (?) request - let's go with `SADD`-like complexity in this (and other variadics that need updates). So something like: "O(1) to set a single field. When called with multiple arguments, O(N) where N is the number of fields to set."

Reply:
changed. O(n) for field/value pairs. It's a little weird, but I think it's the best language now.